payment/locale/fr/LC_MESSAGES/django.po

```diff
@@ -125,14 +125,14 @@ msgid ""
 msgstr ""
```

Comment:
The proctored exam is managed by our provider ProctorU. The exam will only visible "
 "for students who have chosen the certified/verified path"